### PR TITLE
LPS-163700 | Test Automation | Master

### PR DESCRIPTION
**Pull request by thaiscabrals**
[FormsSubmissions#CanSeePartialResultsButtonWithLimitsOneSubmissionPerUser](https://issues.liferay.com/browse/LPS-163700)

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsSubmissions.testcase [hunk 1]
```diff
@@ -778,6 +778,71 @@ definition {
 			successPageTitle = "Thank you.");
 	}
 
+	@description = " Verify that a user on a form that limits one submission per user can see partial results when accessing the forms link."
```
Comment (Luizcarlosqueiroz) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsSubmissions.testcase:781]: Hey @thaiscabrals!
Could you add the related story LPS? This is something we can discuss with the team to start doing (when it's possible)

**Review by Luizcarlosqueiroz — APPROVED** [review]
LGTM! :tada:

**Comment by thaiscabrals** [comment]
ci:test:sf

**Review by Luizcarlosqueiroz — APPROVED** [review]

**Comment by thaiscabrals** [comment]
ci:forward

**Pull request closed without merging**